management/operatingsystemimage/entities.go

```diff
@@ -0,0 +1,29 @@
+package operatingsystemimage
+
+import (
+	"encoding/xml"
+
+	"github.com/MSOpenTech/azure-sdk-for-go/management"
+)
+
+//ImageClient is used to manage operations on Azure Locations
+type OsImageClient struct {
+	client management.Client
+}
+
+type imageList struct {
+	XMLName  xml.Name  `xml:"Images"`
+	Xmlns    string    `xml:"xmlns,attr"`
+	OSImages []OSImage `xml:"OSImage"`
+}
+
+type OSImage struct {
+	Category        string
+	Label           string
+	LogicalSizeInGB string
```

Comment:
@paulmey equivalent of double is `float` in go and that's an alias for `float64` so we must be using `float` for other float32s as well